websocket.go

```diff
@@ -12,6 +13,13 @@ const writeWait = 10 * time.Second
 const pongWait = 60 * time.Second
 const pingPeriod = (pongWait * 9) / 10
 
+const (
+	CloseNormalClosure   = 1000
+	ClosePolicyViolation = 1008
+	CloseInternalError   = 1011
+	CloseRoomNotFound    = 4004 // Custom close code for room not found
+)
```

Comment:
Apart from CloseRoomNotFound, the other codes don't seem to be used anywhere.

Reply:
removed